packages/@aws-cdk/aws-codepipeline-actions/README.md

```diff
@@ -945,6 +945,28 @@ new codepipeline.Pipeline(this, 'Pipeline', {
 });
 ```
 
+### Elastic Beanstalk Deployment
+
+To deploy an Elastic Beanstalk Application in CodePipeline:
+
+```ts
+const sourceOutput = new codepipeline.Artifact();
+const targetBucket = new s3.Bucket(this, 'MyBucket');
+
+const pipeline = new codepipeline.Pipeline(this, 'MyPipeline');
+const deployAction = mew codepipeline_actions.ElasticBeanstalkDeployAction({
```

Comment:
```suggestion
const deployAction = new codepipeline_actions.ElasticBeanstalkDeployAction({
```

Reply:
If I ever thought that I could type a damn thing without spell check... I've been corrected. 😬